src/directions/main.ts

```diff
@@ -49,13 +50,13 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
   protected onDragMoveHandler: (e: MapMouseEvent) => void;
   protected onDragUpHandler: (e: MapMouseEvent) => void;
   protected onClickHandler: (e: MapMouseEvent) => void;
+  protected liveRefreshHandler: (e: MapMouseEvent | MapTouchEvent) => void;
```

Comment:
A good point on `MapTouchEvent`. Maybe makes sense to change accordingly the other occurances?

Reply:
Added `MapTouchEvent` to pre-existing handlers